tests/features/content_types/content_type_administration.feature

```diff
@@ -0,0 +1,48 @@
+@api
+Feature: Content type administration features
+  In order to easily structure the content of the European Commission
+  As an administrator
+  I want to be able to create, customize and delete content types
+
+  Background:
+    Given I am logged in as a user with the 'administrator' role
+
+  Scenario: As administrator, I create a custom content type translated through
```

Comment:
All scenarios for this feature are for Administrator. I think it's not necessary to specify again here.